tutor/src/screens/assignment-edit/clone/editor.js

```diff
@@ -102,7 +103,7 @@ class Editor extends React.Component {
           <div>
             <Button
               variant="default"
-              onClick={onHide}
+              onClick={() => onHide({ shouldRefreshPastPlans: false })}
               size="lg"
             >
             Cancel
```

Comment:
I had an alternative thought here.  How about we hook into the `assignment-edit/ux` `saveAndCopyPlan ` https://github.com/openstax/tutor-js/blob/master/tutor/src/screens/assignment-edit/ux.js#L444 if we check `this.plan.isClone` https://github.com/openstax/tutor-js/blob/master/tutor/src/models/task-plans/teacher/plan.js#L163 and if that's set, we remove it from `this.course.pastTaskPlans`

That would remove the past task plan immediately and save us a fetch. 

What do you think?

Reply:
oh that works. great idea @nathanstitt .